nucypher/blockchain/eth/deployers.py

```diff
@@ -870,11 +870,20 @@ def __init__(self, *args, **kwargs):
                                                                     contract_name=policy_contract_name,
                                                                     proxy_name=policy_proxy_name)
 
+        worklock_name = WorklockDeployer.contract_name
+        try:
+            self.worklock_contract = self.blockchain.get_contract_by_name(registry=self.registry,
+                                                                          contract_name=worklock_name)
+        except BaseContractRegistry.UnknownContract:
+            self.worklock_contract = None
```

Comment:
Does this need follow once deployed on mainnet?

Reply:
I thought to use this moment more for testnets without worklock, so depending on that